src/main/java/com/jfrog/ide/common/go/GoScanWorkspaceCreator.java

```diff
@@ -0,0 +1,79 @@
+package com.jfrog.ide.common.go;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.jfrog.build.api.util.Log;
+import org.jfrog.build.extractor.go.GoDriver;
+
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.FileVisitor;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.Map;
+
+/**
+ * This FileVisitor copies all go.mod and *.go files from the input source directory to the input target directory.
+ * For all go.mod files, replaces relative paths to absolute.
```

Comment:
Let's also add to this description, that functionality is needed. so that we can calculate the go depedencies tree on a copy of the original code project, rather than on the original one, in order to avoid changing it.